community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        [  
+          {
+            "apiVersion": "pingcap.com/v1alpha1",
+            "kind": "TidbCluster",
+            "metadata": {
+              "name": "demo",
+              "namespace": "tidb1"
+              },
+            "spec": {
+              "pd": {
+                  "image": "pingcap/pd:v2.1.0",
+                  "imagePullPolicy": "IfNotPresent",
+                  "limits": {},
+                  "nodeSelectorRequired": "true",
+                  "replicas": "3",
+                  "requests": {
+                    "storage": "5Gi"
+                  },
+                  "storageClassName": "pd-ssd"
+                },
+              "pvReclaimPolicy": "Retain",
+              "schedulerName": "tidb-scheduler",
+              "services": {
```

Comment:
Lis list is not formatted correctly, add a '[' & ']' to enclose.

Reply:
@ceohockey60 This should be updated. From our tests, using a singleton list here allowed for the CR to be picked up. Let us know if you disagree

Reply:
This is referring to the "services" list, correct?

Reply:
Yup!

Reply:
Cool, just made the fix.